graphql/src/main/java/io/jmix/graphql/datafetcher/EntityQueryDataFetcher.java

```diff
@@ -247,4 +302,51 @@ protected LogicalCondition createCondition(Object filter) {
 
         return query;
     }
+
```

Comment:
Methods below (getCustom***Loaders) should be extracted to separate class, together with logic of getting loader bean for exact entity class.